src/Libraries/CoreNodeModels/Input/DateTime.cs

```diff
@@ -41,7 +41,7 @@ public override NodeInputData InputData
                     Type = NodeInputData.getNodeInputTypeFromType(typeof(System.DateTime)),
                     Description = this.Description,
                     //format dateTime with swagger spec in mind:  ISO 8601.
-                    Value = Value.ToString("s",CultureInfo.InvariantCulture),
+                    Value = Value.ToUniversalTime().ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fff'Z'"),
```

Comment:
did you check AVP format?

Reply:
Is this better than using the function built specifically for this?

    Value = XmlConvert.ToString(Value, XmlDateTimeSerializationMode.Utc);

Reply:
@ramramps I did check that AVP is using UTC, so the round trip should be good. @gregmarr @ColinDayOrg  I was a bit hesitate to use XML serialization code but I just found there is the easiest way using `ToString("O")`
```
The "O" or "o" standard format specifier (and the "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffffffK" custom format string) takes advantage of the three ways that ISO 8601 represents time zone information to preserve the Kind property of DateTime values:
The time zone component of DateTimeKind.Utc date and time values uses "Z" (which stands for zero offset) to represent UTC.
```